portiaui/app/adapters/application.js

```diff
@@ -377,7 +378,7 @@ export default DS.JSONAPIAdapter.extend(UrlTemplates, {
             promise.finally(() => {
                 const project = this.get('uiState.models.project');
                 if (project) {
-                    project.markChanged();
+                    this.set('changes._changes', true);
```

Comment:
Don't change the 'private' interface. Instead, expose a public method which marks changes as `true` or change `_changes` into a public `changes`.